lenskit-core/src/main/java/org/lenskit/data/entities/CommonAttributes.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * LensKit, an open source recommender systems toolkit.
+ * Copyright 2010-2014 LensKit Contributors.  See CONTRIBUTORS.md.
+ * Work on LensKit has been funded by the National Science Foundation under
+ * grants IIS 05-34939, 08-08692, 08-12148, and 10-17697.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package org.lenskit.data.entities;
+
+/**
+ * Definitions and utilities for common fields.
+ */
+public final class CommonAttributes {
```

Comment:
I'm worried that we are moving towards a pattern where this class needs to be updated every time we make a new event type. ID, user, and item are reasonable because they very crosscutting in our domain. Timestamp is a little sketchier, but I could see it being used a lot. Name, Rating, and Count seem more specific to only one entity type. I'm not sure I want those in a central location.